uiowa/uiowa
# Update Grad program list

Resolves #6083 

# How to test

```
ddev blt ds --site grad.uiowa.edu && ddev drush @grad.local cr && ddev drush @grad.local uli node/add/thesis_defense
```
In the "Degree major" drop-down, check for:
- "Criminology" (newly added PhD option)
- "Mass Communication" (name change from "Mass Communications"
- Consult the list of changes to do any additional spot checking you feel like: https://github.com/uiowa/uiowa/pull/6123/files. You are looking for items is `'PhD'` listed under `'objectives'`.
